packages/js/src/indexables-page/indexables-page.js

```diff
@@ -74,20 +75,34 @@ const leastLinkedOutro = addLinkToString(
 	"https://www.yoast.com"
 );
 
-const mostLinkedIntro = addLinkToString(
-	// translators: %1$s and %2$s are replaced by opening and closing anchor tags.
-	sprintf(
+const mostLinkedIntro = <Fragment>
+	{
 		__(
-			"The content below is supposed to be your cornerstone content: the most important and extensive articles on your site. " +
-			"Make sure to mark this content as cornerstone content to get all bullets green. " +
-			"%1$sLearn more about cornerstone content%2$s.",
-			"wordpress-seo"
-		),
-		"<a>",
-		"</a>"
-	),
-	"https://www.yoast.com"
-);
+			"The content below is supposed to be your cornerstone content: the most important and extensive articles on your site. ", "wordpress-seo"
+		)
+	}
+	{
+		createInterpolateElement(
+			// translators: %1$s and %2$s are replaced by opening and closing span tags.
+			// %3$s is replaced by the StarIcon component.
+			// %4$s and %5$s are replaced by opening and closing anchor tags.
+			sprintf(
+				__( "Make sure to mark this content as cornerstone content %1$s(%3$s)%2$s. %4$sLearn more about cornerstone content%5$s.", "wordpress-seo" ),
+				"<span>",
+				"</span>",
+				"<StarIcon />",
+				"<a>",
+				"</a>"
+			),
+			{
+				span: <span className="yst-whitespace-nowrap" />,
+				// eslint-disable-next-line jsx-a11y/anchor-has-content
+				a: <a id="kek" href="https://www.yoast.com" target="_blank" rel="noopener noreferrer" />,
```

Comment:
kek 🤣 
```suggestion
				a: <a id="indexables-page-cornerstone-link" href="https://www.yoast.com" target="_blank" rel="noopener noreferrer" />,
```
Also, did we get a shortlink for this?

Reply:
🤣 that slipped in... 
And no, we currently need *all* the shortlinks!